### ISSUE TITLE
Ignored `SequenceExpression` argument not correctly printed

### ISSUE THREAD

**Issue by fisker** [prettier/prettier#13737]
**Prettier pr-13736**
[Playground link](https://deploy-preview-13736--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuE04EEBOBzArgWwRgAoAdKAAgHoqKAHTOGGASzkwFoXsoJHyKxAAwAaCgEYAlOUkgRqOq2gBnZKACGmTBADuABU0JVKdQBsd6gJ6r5AI0zqwAayYBldYQAyLKHGQAzM2U4OwdnNzpHH2xkGExcEJA4fFs4ABM09M91KDx1bDgAMT58dWZo5BB1XBgIORAACxh8UwB1BpZ4ZUiwOFcjTpYAN07LSrBlGxAfYMwYPQdsUoCgxIArZQAPV2jTOABFXAh4FdNg+UjMWcrbdVTTeoYfGFaWNJgG5AAOURAGCGCrQcdEqDDgsyGfnkAEcjvAFhA6MYqsoOL50ul6oxYSxGAt8sskIEzolgvgWLF4qTdgc4X4iat5DA7q93p8kAAmJkOFimaIAYQg+EJVVMD3kuGCABU7sZiecQEMEgBJKCZWCuMCYFiKdBq1wwSx7U7BAC+pqAA)

```sh
--parser babel
```

**Input:**

```jsx
oneArgument(
 // prettier-ignore
 (0, 1)
)
```

**Output:**

```jsx
oneArgument(
  // prettier-ignore
  0, 1,
);

```

**Expected behavior:**

Should keep parenthesized.

**Comment by fisker** [comment]
Assignment

**Prettier pr-13736**
[Playground link](https://deploy-preview-13736--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEBDAvACgDpQAQD0BeADgE5wwwCWcZAtNQOZQQW56YAMANHgIwBKXIJA8QEEjWgBnZKFRkyEAO4AFRQjkpUAGxWoAnnPEAjMqjABrSgGVUAWzgAZalDjIAZnplwzF6zsSSzcmZBgyAFc-EDgHUzgAE0Sk51QoJkjUJjgAMTYHVCpQ5DRImAgxEAALGAddAHVq6ngZYLA4Wy0W6gA3FsNSsBkTEDdfMhg1CyZCrx8YgCsZAA9bUN04AEVIiHh53V9xYLIJ0tNUBN0q8jcYBupEmGrkAA5eEHIIXwaLElLyHAJr0POIAI67eDTSTaNAyejuJJJKoUCHUCjTbJzJDeQ4xXwOajhKL4jbbSEeHELcQwS4PJ4vJAAJhpFmoulCAGEIA5sWhdNdxJFfAAVS7aXFHEC9aIASSgKVgtjAZGoUgAggrbDBDJsDr4AL4GoA)

```sh
--parser babel
```

**Input:**

```jsx
a=(
 // prettier-ignore
 (0, 1)
)
```

**Output:**

```jsx
a =
  // prettier-ignore
  0, 1;

```

**Second Output:**

```jsx
(a =
  // prettier-ignore
  0),
  1;

```

**Comment by fisker** [comment]
Fixed by #14406
